### PR TITLE
process in collect_in_cluster

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: src/patch/peripheral.rs [hunk 1]
```diff
@@ -1365,7 +1381,7 @@ fn collect_in_cluster(
 
     for (rspec, rmod) in cmod {
         let rspec = rspec.str()?;
-        if rspec == "description" || rspec == "dimIncrement" {
+        if rspec == "description" || rspec == "dimIncrement" || rspec.starts_with('_') {
```
Comment (Emilgardis) [inline src/patch/peripheral.rs:1384]: why is this needed?

Comment (burrbull) [reply src/patch/peripheral.rs]: Is now better?

**Review by Emilgardis — APPROVED** [review]
